CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,159 @@
+   * discriminatory jokes and language
+   * posting sexually explicit or violent material
+   * posting (or threatening to post) other people's personally identifying
+     information ("doxing")
+   * personal insults, especially those using racist or sexist terms
+   * unwelcome sexual attention
+   * advocating for, or encouraging, any of the above behavior
+   * repeated harassment of others. In general, if someone asks you to stop,
+     then stop
+
+6. **Moderate your expectations**. Many in our community volunteer their time.
+   They are probably not purposefully ignoring issues, refusing to engage in
+   discussion, avoiding features, etc. but often just unavailable.
+
+7. **When we disagree, try to understand why**. Disagreements, both social and
+   technical, happen all the time and MDAnalysis is no exception. It is important
+   that we resolve disagreements and differing views constructively. Remember
+   that we’re different. The strength of MDAnalysis comes from its varied community
+   that includes people from a wide range of backgrounds. Different people have
+   different perspectives on issues. Being unable to understand why someone
+   holds a viewpoint doesn’t mean they’re wrong. Don’t forget that it is human
+   to err and blaming each other doesn’t get us anywhere. Instead, focus on
+   helping to resolve issues and learning from mistakes.
+
+8. **A simple apology can go a long way**. It can often de-escalate a situation,
+   and telling someone that you are sorry is act of empathy that doesn’t
+   automatically imply an admission of guilt.
+
+# Reporting
+
+If someone makes you or any other contributor feel unsafe or unwelcome, please
+report this in a timely manner. Code of conduct violations reduce the value of
+the community for everyone and we take them seriously. All complaints will be
+reviewed and investigated and will result in a response that is deemed necessary
+and appropriate to the circumstances.
+
+You can file a report by emailing
+the [*Conduct-mail*][conduct-mail] or by
+filing out [this form][conduct-form]. The project team is obligated to maintain
+confidentiality with regard to the reporter of an incident.
+
+The online form gives you the option to keep your report anonymous or request
+that we follow up with you directly. While we cannot follow up on an anonymous
+report, we will take appropriate action.
+
+# Enforcement
+
+When a report is sent to us we will reply as soon as possible to confirm receipt;
+we strive to answer in less than 24 hours. We will review the incident and
+determine, to the best of our ability
+
+- what happened
+- whether this event constitutes a code of conduct violation
+- who, if anyone, was at fault
+- whether this is an ongoing situation
+
+This information will be collected in writing. We strive to reach a resolution
+within a week after conformation. Once a resolution has been agreed upon, but before it is
```

Comment:
Still needs fixing:
"within a week *of* *confirmation*" or do you really mean "conformation"?
(Also note the "of" instead of "after")